### PR TITLE
[translation] Drop Python 3.6 support.

### PR DESCRIPTION
**Pull request by wonhyeongseo**
# Description
Closes https://github.com/Azure/azure-sdk-for-python/issues/25167

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/translation/azure-ai-translation-document/README.md [hunk 1]
```diff
@@ -12,11 +12,12 @@ Use the client library for Document Translation to:
 ## _Disclaimer_
 
 _Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+_Python 3.7 or later is required to use this package. For more details, please refer to [Azure SDK for Python version support policy](https://github.com/Azure/azure-sdk-for-python/wiki/Azure-SDKs-Python-version-support-policy)._
```
Comment (kristapratico) [inline sdk/translation/azure-ai-translation-document/README.md:15]: We put the special disclaimer for Python 2.7 to give customers a heads up that they may need to update their code, but we don't need to do the same for dropping Python 3.6. You can remove this comment.

**Review by kristapratico — COMMENTED** [review]
Thanks for your contribution! Once the review comment is resolved I can approve and merge your PR. :)

File: sdk/translation/azure-ai-translation-document/README.md (hunk 1 above)
Comment (wonhyeongseo) [reply sdk/translation/azure-ai-translation-document/README.md]: Thank you so much for your review @kristapratico !
I have updated README.md. Have a wonderful day!

**Review by kristapratico — APPROVED** [review]
Thanks again! :)

**Pull request merged**